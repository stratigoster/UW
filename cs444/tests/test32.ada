PACKAGE foo IS
  one : integer := 1;
  twentyfive : integer := 25;
  type coins is (one,5,10,twentyfive);
--               ^^^
-- can only overload enumeration elements
END foo;
