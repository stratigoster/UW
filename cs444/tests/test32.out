***ERROR***:4:22:	Expected "Id" but found "5"
